examples/storm-kafka-examples/pom.xml

```diff
@@ -24,6 +24,11 @@
         <version>2.0.0-SNAPSHOT</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
+    
+    <properties>
+        <!-- Override with -Dkafka.dependency.scope=compile to generate a jar with dependencies -->
+        <kafka.dependency.scope>provided</kafka.dependency.scope>
```

Comment:
This workaround is necessary because this module is included as a dependency in storm-kafka-client-examples, so we need a normal install to generate a slim jar.